pkg/collector/corechecks/oracle-dbm/config/config.go

```diff
@@ -78,30 +78,31 @@ type CustomQuery struct {
-	Wallet                 string               `yaml:"wallet"`
-	DBM                    bool                 `yaml:"dbm"`
-	Tags                   []string             `yaml:"tags"`
-	LogUnobfuscatedQueries bool                 `yaml:"log_unobfuscated_queries"`
-	ObfuscatorOptions      obfuscate.SQLConfig  `yaml:"obfuscator_options"`
-	InstantClient          bool                 `yaml:"instant_client"`
-	ReportedHostname       string               `yaml:"reported_hostname"`
-	QuerySamples           QuerySamplesConfig   `yaml:"query_samples"`
-	QueryMetrics           QueryMetricsConfig   `yaml:"query_metrics"`
-	SysMetrics             SysMetricsConfig     `yaml:"sysmetrics"`
-	Tablespaces            TablespacesConfig    `yaml:"tablespaces"`
-	ProcessMemory          ProcessMemoryConfig  `yaml:"process_memory"`
-	SharedMemory           SharedMemoryConfig   `yaml:"shared_memory"`
-	ExecutionPlans         ExecutionPlansConfig `yaml:"execution_plans"`
-	AgentSQLTrace          AgentSQLTrace        `yaml:"agent_sql_trace"`
-	CustomQueries          []CustomQuery        `yaml:"custom_queries"`
+	Server                        string               `yaml:"server"`
+	Port                          int                  `yaml:"port"`
+	ServiceName                   string               `yaml:"service_name"`
+	Username                      string               `yaml:"username"`
+	Password                      string               `yaml:"password"`
+	TnsAlias                      string               `yaml:"tns_alias"`
+	TnsAdmin                      string               `yaml:"tns_admin"`
+	Protocol                      string               `yaml:"protocol"`
+	Wallet                        string               `yaml:"wallet"`
+	DBM                           bool                 `yaml:"dbm"`
+	Tags                          []string             `yaml:"tags"`
+	LogUnobfuscatedQueries        bool                 `yaml:"log_unobfuscated_queries"`
+	ObfuscatorOptions             obfuscate.SQLConfig  `yaml:"obfuscator_options"`
+	InstantClient                 bool                 `yaml:"instant_client"`
+	ReportedHostname              string               `yaml:"reported_hostname"`
+	QuerySamples                  QuerySamplesConfig   `yaml:"query_samples"`
+	QueryMetrics                  QueryMetricsConfig   `yaml:"query_metrics"`
+	SysMetrics                    SysMetricsConfig     `yaml:"sysmetrics"`
+	Tablespaces                   TablespacesConfig    `yaml:"tablespaces"`
+	ProcessMemory                 ProcessMemoryConfig  `yaml:"process_memory"`
+	SharedMemory                  SharedMemoryConfig   `yaml:"shared_memory"`
+	ExecutionPlans                ExecutionPlansConfig `yaml:"execution_plans"`
+	AgentSQLTrace                 AgentSQLTrace        `yaml:"agent_sql_trace"`
+	CustomQueries                 []CustomQuery        `yaml:"custom_queries"`
+	NonActivityCollectionInterval int64                `yaml:"non_activity_collection_interval"`
```

Comment:
does it make sense to call this `metric_collection_interval`? or something that describes what its collecting instead of what its not collecting?